src/main/java/com/suchocki/parkingmeter/dao/ParkActionDAOImpl.java

```diff
@@ -0,0 +1,72 @@
+package com.suchocki.parkingmeter.dao;
+
+import java.util.List;
+
+import org.springframework.stereotype.Repository;
+import com.suchocki.parkingmeter.database.FakeDatabaseStub;
+import com.suchocki.parkingmeter.entity.Driver;
+import com.suchocki.parkingmeter.entity.ParkAction;
+
+@Repository
+public class ParkActionDAOImpl implements ParkActionDAO {
+
+	@Override
+	public void save(ParkAction parkAction) {
+		for (ParkAction pAction : FakeDatabaseStub.parkActions) {
```

Comment:
jak to by wyglądało w stylu Javy 8?

Reply:
```
if (FakeDatabaseStub.parkActions.stream().anyMatch(action -> (action.getId() == parkAction.getId()))) {
	update(parkAction);
} else {
	parkAction.setId(FakeDatabaseStub.parkActions.size() + 1);
	FakeDatabaseStub.parkActions.add(parkAction);
}
```
Choć w pierwotnym rozwiązaniu brakuje powyższego _else_.